test/paratii.eth.vouchers.js

```diff
@@ -0,0 +1,36 @@
+import { Paratii } from '../lib/paratii.js'
+import { address, privateKey, voucherCode11, voucherAmount11, voucherAmountInitial11, hashedVoucherCode11 } from './utils.js'
+import { assert } from 'chai'
+
+describe('paratii.eth.vouchers:', function () {
+  let paratii
+
+  beforeEach(async function () {
+    paratii = await new Paratii({
+      provider: 'http://localhost:8545',
+      address: address,
+      privateKey: privateKey
+    })
+    await paratii.eth.deployContracts()
+    let token = await paratii.eth.getContract('ParatiiToken')
+    let vouchers = await paratii.eth.getContract('Vouchers')
+    let addressOfVouchers = vouchers.options.address
+    await token.methods.transfer(addressOfVouchers, voucherAmountInitial11)
+  })
+
+  it('vouchers.add() should work as expected', async function () {
+    let result = await paratii.eth.vouchers.add({ voucherCode: voucherCode11, amount: voucherAmount11 })
+    assert.equal(hashedVoucherCode11, result)

```

Comment:
These tests are fine, but you could also test some of the type checking and error handling